src/zenml/constants.py

```diff
@@ -214,6 +214,7 @@ def handle_int_env_var(var: str, default: int = 0) -> int:
 SECRETS = "/secrets"
 VISUALIZE = "/visualize"
 CODE_REPOSITORIES = "/code_repositories"
+CODE_REFERENCES = "/code_references"
```

Comment:
I have added these endpoints for the sake of hydration since they were based on the same base response model. I already planned an optimization branch that will follow this PR. I will try to optimize this!